lib/foreman_maintain/cli/packages_command.rb

```diff
@@ -55,12 +55,14 @@ def execute
       subcommand 'update', 'Update packages in an unlocked session' do
         interactive_option(['assumeyes'])
         parameter '[PACKAGES] ...', 'packages to update', :attribute_name => :packages
+        option '--cache', :flag, 'Downloads and caches package updates only', :default => false
```

Comment:
Should this be `--downloadonly` to match DNF naming?

Reply:
> Should this be `--downloadonly` to match DNF naming?

:man_facepalming: Yes, yes it should.